hazenlib/spatial_resolution.py

```diff
@@ -461,7 +466,7 @@ def calculate_mtf_for_edge(dicom, edge, report_path=False):
         axes[9].set_title('line spread function')
         axes[9].plot(lsf)
         axes[10].set_title('normalised MTF')
-        axes[10].plot(norm_mtf)
+        axes[10].plot(freqs[mask],norm_mtf[mask])
```

Comment:
[Update--see comment on line 430 for possible resolution]

I can't figure out the units on this axis. Are they pixels^(-1)? The MTF graph for a 512 pixel image (tests\data\resolution\RESOLUTION\ax_512_resolution.dcm) and the 256 pixel image (tests\data\resolution\eastkent\256_sag.IMA) are very similar.
512:
![image](https://user-images.githubusercontent.com/87647715/137476072-50a5540c-2931-4698-b822-eb0f544f2d72.png)
256:
![image](https://user-images.githubusercontent.com/87647715/137476142-8a579136-99bf-4df4-9dd3-c24e2ac61585.png)

The actual spatial resolution of the 512 image is around 1 line pair /mm and for the 256 image it's about 2 line pairs /mm as can be seen on the comb inserts.